mysql-demo.yml

```diff
@@ -68,7 +71,7 @@ services:
       - XPACK_REPORTING_ENABLED=false 
       - XPACK_MONITORING_ENABLED=false
     volumes: 
-      - ./pipelines/mysql_pipelines.conf:/usr/share/logstash/pipeline/logstash.conf
+      - ./pipelines/mysql_sqs_pipeline.conf:/usr/share/logstash/pipeline/logstash.conf
```

Comment:
dont change here ....create a new file call mysql-sqs-demo.yml and copy paste content